src/backend/base/langflow/custom/custom_component/component.py

```diff
@@ -91,9 +91,9 @@ def __init__(self, **kwargs) -> None:
         self.__inputs = inputs
         self.__config = config
         self._reset_all_output_values()
-        if FEATURE_FLAGS.add_toolkit_output and hasattr(self, "_append_tool_output"):
-            self._append_tool_output()
         super().__init__(**config)
+        if (FEATURE_FLAGS.add_toolkit_output) and hasattr(self, "_append_tool_output") and self.add_tool_output:
```

Comment:
It is weird that we have 3 separate flags that mean about the same thing. Not a blocker for this, but will want to clean this up when we can.

Reply:
ok! Sure
Once the feature flag is removed we can simplify this just to
if self.add_tool_output:
.....